src/pages/billing/Project.js

```diff
@@ -17,18 +17,35 @@ import { withErrorReporting } from 'src/libs/error'
 import Events from 'src/libs/events'
 import { FormLabel } from 'src/libs/forms'
 import * as Nav from 'src/libs/nav'
+import { contactUsActive } from 'src/libs/state'
 import * as StateHistory from 'src/libs/state-history'
+import { topBarHeight } from 'src/libs/style'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 import { billingRoles } from 'src/pages/billing/List'
 
 
 const workspaceLastModifiedWidth = 150
 const workspaceExpandIconSize = 20
+const billingAccountIconSize = 16
 
-const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort, onSort }) => {
+const BillingAccountIcon = {
+  updating: { shape: 'sync', color: colors.warning() },
+  done: { shape: 'check', color: colors.accent() },
+  error: { shape: 'warning-standard', color: colors.danger() }
+}
+
+const getBillingAccountIcon = status => {
+  const { shape, color } = BillingAccountIcon[status]
+  return icon(shape, { size: billingAccountIconSize, color })
+}
+
+const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ needsStatusColumn, sort, onSort }) => {
```

Comment:
Sorry if we've already covered this, but why does this need to be memoized? Is there something about computing it that's super expensive, or that we expect to have it called over and over again?